### PR TITLE
[zh] Resync tasks pages (task-3)

### PR DESCRIPTION
**Pull request by tengqm**
Related: #34221

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/reconfigure-kubelet.md [hunk 1]
```diff
@@ -84,4 +85,6 @@ satisfying the [Kubernetes version skew policy](/releases/version-skew-policy/).
 -->
 请注意，从 v1.24 开始 `DynamicKubeletConfig` 特性门控无法在 kubelet 上设置，
 因为不会生效。在 v1.26 之前 API 服务器和控制器管理器不会移除该特性门控。
-这是专为控制面支持有旧版本 kubelet 的节点以及满足 [Kubernetes 版本偏差策略](/releases/version-skew-policy/)。
\ No newline at end of file
+这是专为控制面支持有旧版本 kubelet 的节点以及满足
+[Kubernetes 版本偏差策略](/releases/version-skew-policy/)。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/reconfigure-kubelet.md:89]:
```suggestion
[Kubernetes 版本偏差策略](/zh-cn/releases/version-skew-policy/)。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]
found a nit~

File: content/zh-cn/docs/tasks/administer-cluster/reconfigure-kubelet.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/tasks/administer-cluster/reconfigure-kubelet.md]: done.

**Pull request merged**